content/02.delete-me.md

```diff
@@ -192,14 +192,23 @@ White background specified to serve as a backdrop for transparent sections of th
 Table: A table with a top caption and specified relative column widths.
 {#tbl:bowling-scores}
 
-|         | Digits 1-33                        | Digits 34-66                      | Digits 67-99                      | Ref.                        |
-|:--------|:-----------------------------------|:----------------------------------|:----------------------------------|:----------------------------|
-| pi | 3.14159265358979323846264338327950 | 288419716939937510582097494459230 | 781640628620899862803482534211706 | [`piday.org`](https://www.piday.org/million/) |
-| e  | 2.71828182845904523536028747135266 | 249775724709369995957496696762772 | 407663035354759457138217852516642 | [`nasa.gov`](https://apod.nasa.gov/htmltest/gifcity/e.2mil) |
+|         | Digits 1-33                        | Digits 34-66                      | Digits 67-99                      | Ref.                                                        |
+|:--------|:-----------------------------------|:----------------------------------|:----------------------------------|:------------------------------------------------------------|
+| pi      | 3.14159265358979323846264338327950 | 288419716939937510582097494459230 | 781640628620899862803482534211706 | [`piday.org`](https://www.piday.org/million/)               |
+| e       | 2.71828182845904523536028747135266 | 249775724709369995957496696762772 | 407663035354759457138217852516642 | [`nasa.gov`](https://apod.nasa.gov/htmltest/gifcity/e.2mil) |
```

Comment:
Do you think all this white-space padding is helpful? Oftentimes, I think wide markdown tables are best in compact format (one space like `| pi | 3.14159265358979323846264338327950 |`). The cells are not aligned in a text editor, but since most text editors wrap long lines, they won't be aligned anyways.

Would it make sense to keep non-compact spacing in table below, but switch to compact in the very wide table here?

Reply:
I just thought it looked ugly personally. I usually view my files unwrapped, so it helps me.

I made sure there was an example of a table without exact spacing, so they know that it's not a requirement.